.github/workflows/ci.yml

```diff
@@ -551,6 +551,50 @@ jobs:
           path: ${{ env.UV_WORKSPACE }}/target/debug/uv.exe
           retention-days: 1
 
+  build-binary-freebsd:
+    needs: determine_changes
+    timeout-minutes: 10
+    if: ${{ github.repository == 'astral-sh/uv' && (needs.determine_changes.outputs.code == 'true' || github.ref == 'refs/heads/main') }}
+    runs-on:
+      labels: ubuntu-latest
+    name: "build binary | freebsd"
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: Swatinem/rust-cache@v2
+      - name: "Cross build"
+        run: |
+          # Install cross from `freebsd-firecracker`
+          wget -q -O cross https://github.com/acj/freebsd-firecracker/releases/download/v0.0.10/cross
+          chmod +x cross
+          mv cross /usr/local/bin/cross
+
+          cross build --target x86_64-unknown-freebsd
+
+      - name: Test in Firecracker VM
+        uses: acj/freebsd-firecracker-action@v0.2.0
+        with:
+          verbose: false
+          pre-run: |
+            include_path="$(mktemp)"
+            cat <<EOF > $include_path
+            target
+            target/x86_64-unknown-freebsd
+            target/x86_64-unknown-freebsd/debug
+            target/x86_64-unknown-freebsd/debug/uv
+            EOF
```

Comment:
Can you add a comment why we're listing more than one path here? It's not apparent to me from the rsync man page

Reply:
I don't understand either. I had to copy this from rbspy — it didn't work with just the last path. I guess the exclude `*` overrides the includes unless we provide each directory?

Reply:
just that as inline comment would be helpful for when we need to touch that job again